charts/vmclarity/templates/apiserver/service-headless.yaml

```diff
@@ -0,0 +1,21 @@
+{{/*
+Headless Service does not include a ClusterIP so that the DNS record includes
+all Pod addresses. This makes it possible for the API Gateway to load balance
+all replicas instead of relying on kube-proxy to do it.
+*/}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ printf "%s-apiserver-hl" (include "vmclarity.names.fullname" .) | trunc 63 | trimSuffix "-" }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "vmclarity.labels.standard" . | nindent 4 }}
```

Comment:
I think we must have a `vmclarity-apiserver.labels.standard` for example, here you are missing the `app.kubernetes.io/component: apiserver` for example.